src/react/index.ts

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {provideReactWrapper} from '@microsoft/fast-react-wrapper';
+import {provideFASTDesignSystem} from '@microsoft/fast-components';
+
+import {
+	vsCodeButton,
+	vsCodeCheckbox,
+	vsCodeBadge,
+	vsCodeDivider,
+	vsCodeLink,
+	vsCodeProgressRing,
+	vsCodeTag,
+	vsCodeTextArea,
+	vsCodeTextField,
+	vsCodeOption,
+	vsCodeDropdown,
+	vsCodePanels,
+	vsCodePanelTab,
+	vsCodePanelView,
+	vsCodeRadio,
+	vsCodeRadioGroup,
+	vsCodeDataGrid,
+	vsCodeDataGridCell,
+	vsCodeDataGridRow,
+} from '..';
+
+const {wrap} = provideReactWrapper(React, provideFASTDesignSystem());
```

Comment:
Update to use `provideVSCodeDesignsystem` instead of `provideFASTDesignSystem`.

Reply:
Resolved by 82e710a8991eebce74fdb702c03f81171c6b684d